Dodge.js

```diff
@@ -1,178 +1,201 @@
 import { StatusBar } from 'expo-status-bar';
+import React, { Component, PureComponent } from 'react';
 import { StyleSheet, Text, View, Dimensions, Button, Image } from 'react-native';
-import { Knight, Circle, Brick, Grenade, MoveKnight, Shrapnel, counter} from "./Entities.js";
+import {
+  Knight, Circle, Brick, Grenade, MoveKnight,
+  Shrapnel, Water
+} from "./Entities.js";
 import { GameEngine } from "react-native-game-engine";
+import Matter from "matter-js";
 import AxisPad from 'react-native-axis-pad';
 ////this looks interesting for the attacker placing/throwing projectiles:
```

Comment:
This is really a note-to-self more than a comment.

Some inline TODOs are ok, but they're easy to get lost. IMO, you should really only use them to mean "this doesn't need to happen, but if I'm changing this section for some other reason, here's what I might do differently".

Tracking ideas is hard! I personally think one of the most important things is to be able to see all your changes/ideas in one place, so you can decide what's important and what to work on next. The list can contain both big ideas "Design stat system" and small ones "Fix the way we calculate damage in Dodge.js on line 73".

I'd either add a notes.txt file to the repository (so it can live and update with the code). Or use the google doc (and make a "Nolan Code TODO" section at the top).

[There are a million TODO apps and project management tools to help you with this, including tools which will look through your code for TODOs and present them to you in one place. I don't know of one right method.]